grpc/src/test/java/com/lightstep/tracer/shared/GrpcAbstractTracerTest.java

```diff
@@ -19,22 +21,13 @@
 import static com.lightstep.tracer.shared.Options.VERBOSITY_FIRST_ERROR_ONLY;
 import static com.lightstep.tracer.shared.Options.VERBOSITY_INFO;
 import static com.lightstep.tracer.shared.Options.VERBOSITY_NONE;
-import static com.lightstep.tracer.shared.TextMapPropagator.FIELD_NAME_SAMPLED;
-import static com.lightstep.tracer.shared.TextMapPropagator.FIELD_NAME_SPAN_ID;
-import static com.lightstep.tracer.shared.TextMapPropagator.FIELD_NAME_TRACE_ID;
-import static com.lightstep.tracer.shared.TextMapPropagator.PREFIX_BAGGAGE;
-import static io.opentracing.propagation.Format.Builtin.BINARY;
-import static io.opentracing.propagation.Format.Builtin.HTTP_HEADERS;
-import static io.opentracing.propagation.Format.Builtin.TEXT_MAP;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyZeroInteractions;
 
 @RunWith(MockitoJUnitRunner.class)
-public class AbstractTracerTest {
+public class GrpcAbstractTracerTest {
```

Comment:
Not sure there is anything GRPC-specific in these tests? I think `AbstractTracerTest` is appropriate.

Reply:
Ya, What really needs to happen is that this test is split between `GrpcCollectorClient` specific tests and `AbstractTracer` specific tests. Where a test client is bound. I was being lazy here. I feel like it might be better to take a pass a better testing either here or in a separate PR.